src/components/views/LocalModList.vue

```diff
@@ -116,78 +116,28 @@
-                            </template>
-                            <template v-else>
-                                <strike class='selectable'>{{key.getDisplayName()}} <span class="card-byline">by {{key.getAuthorName()}}</span></strike>
-                            </template>
-                        </span>
-                    </span>
-                </template>
-                <template v-slot:other-icons>
-                    <!-- Show update and missing dependency icons -->
-                    <span class='card-header-icon has-tooltip-left'
-                          data-tooltip='An update is available' v-if="!isLatest(key)">
-											<i class='fas fa-cloud-upload-alt'></i>
-										</span>
-                    <span class='card-header-icon has-tooltip-left'
-                          :data-tooltip="`Missing ${getMissingDependencies(key).length} dependencies`"
-                          v-if="getMissingDependencies(key).length > 0">
-											<i class='fas fa-exclamation-circle'></i>
-										</span>
-                </template>
-                <a class='card-footer-item'
-                   @click="uninstallModRequireConfirmation(key)">Uninstall</a>
-                <template>
-                    <a class='card-footer-item' @click="disableModRequireConfirmation(key)"
-                       v-if="key.enabled">Disable</a>
-                    <a class='card-footer-item' @click="enableMod(key)" v-else>Enable</a>
-                </template>
-                <a class='card-footer-item' @click="viewDependencyList(key)">View associated</a>
-                <Link :url="`${key.getWebsiteUrl()}${key.getVersionNumber().toString()}`"
-                      :target="'external'"
-                      class="card-footer-item">
-                        <i class='fas fa-code-branch'>&nbsp;&nbsp;</i>
-                        {{key.getVersionNumber().toString()}}
-                </Link>
-                <a class='card-footer-item' v-if="!isLatest(key)" @click="updateMod(key)">Update</a>
-                <a class='card-footer-item' v-if="getMissingDependencies(key).length > 0"
-                   @click="downloadDependency(getMissingDependencies(key)[0])">
-                    Download dependency
-                </a>
-            </expandable-card>
-        </div>
+        <draggable ref="draggable" v-model='sortableModList' group="mods"
+                   animation="200"
+                   @start="startSort()" 
+                   @end="endSort()">
+                    <local-mod-card v-for='(key, index) in sortableModList' :key="'local-' + key.getName()"
+                                    :id="index"
+                                    :manifest="key"
+                                    :missingDependencies="getMissingDependencies(key)"
+                                    :funkyMode="settings.funkyModeEnabled"
+                                    @update:enabled="toggleMod(key, $event)"
```

Comment:
Could you change the event name to update-enabled to be more inline with other event names?